resources/ext/ezbake.conf

```diff
@@ -2,8 +2,8 @@ ezbake: {
   pe: {}
   foss: {
     redhat: { dependencies: ["puppet >= 3.7.3", "puppet < 5.0.0"],
```

Comment:
@haus did you mean "No, _puppet_ provides puppet of 4.0.0 or higher, which redhat handles just fine. Debian can't cope in the same way." ... that is, requiring puppet is all that is needed for rhel for now?